client/src/app/components/Navbar/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { grey } from '@material-ui/core/colors';
+import { AppBar, IconButton, Toolbar, Typography } from '@material-ui/core';
```

Comment:
The directions aren't easy for me to understand yet, but it would appear that they encourage to not do this, but rather import each one directly?

https://material-ui.com/guides/minimizing-bundle-size/#option-1